src/qt/bitcoingui.cpp

```diff
@@ -321,10 +321,8 @@ void BitcoinGUI::createActions()
     optionsAction->setMenuRole(QAction::PreferencesRole);
     openConfigAction = new QAction(tr("Open config &file..."), this);
     optionsAction->setToolTip(tr("Open the config file in your standard editor"));
-    openConfigAction->setMenuRole(QAction::PreferencesRole);
     researcherAction = new QAction(tr("&Researcher Wizard..."), this);
     researcherAction->setToolTip(tr("Open BOINC and beacon settings for Gridcoin"));
-    researcherAction->setMenuRole(QAction::PreferencesRole);
     toggleHideAction = new QAction(tr("&Show / Hide"), this);
```

Comment:
Please put a comment in the code on this, so that future folks that are modifying this area don't undo the fix by mistake. Thanks.

Reply:
If you are busy. I will do it.

Reply:
It's fine thanks, done.